components/LoanReturn/DrawerContent.js

```diff
@@ -0,0 +1,229 @@
+import { Grid, Paper, Typography, Avatar } from '@mui/material';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const DrawerContent = ({ item }) => {
+  const isMobile = useMediaQuery('(max-width: 600px)');
+  const isPC = useMediaQuery('(min-width: 1000px)');
+
+  return (
+    <div>
+      <Paper
+        variant='elevation'
+        style={{
+          padding: '8px',
+          paddingLeft: '16px',
+          paddingRight: '16px',
+          paddingBottom: '16px',
+          borderRadius: '0px',
+        }}
+      >
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'center',
+            marginTop: '15px',
+          }}
+        >
+          <Avatar
+            alt='new-item'
+            src={item.imgpic}
+            style={{ width: '70px', height: '70px' }} // Increase the size
+          />
+        </div>
+      </Paper>
+      <Paper
+        variant='elevation'
+        style={{
+          padding: '8px',
+          paddingLeft: '16px',
+          paddingRight: '16px',
+          borderRadius: '0px',
+        }}
+      >
+        <Grid
+          container
+          spacing={2}
+          sx={{
+            width: isMobile ? '100%' : isPC ? '50%' : '70%',
+          }}
+        >
+          <Grid item xs={6}>
+            <Typography variant='body2' align='left'>
+              Item Name:
+            </Typography>
+          </Grid>
+          <Grid item xs={6}>
+            <Typography variant='body2' align='right'>
+              {item.name}
+            </Typography>
+          </Grid>
+        </Grid>
+      </Paper>
+
+      <Paper
+        variant='elevation'
+        style={{
+          padding: '8px',
+          paddingLeft: '16px',
+          paddingRight: '16px',
+          borderRadius: '0px',
+        }}
+      >
+        <Grid
+          container
+          spacing={2}
+          sx={{
+            width: isMobile ? '100%' : isPC ? '50%' : '70%',
```

Comment:
seeing this style being repeated multiple times 

we have a `dynamic-width` class in `inventory/src/scss/inventoryBase.scss` which u can import. Wouild be nice to standardise the responsive width. But not sure how it will look like, not pertinent now